### PR TITLE
chore: storybook and chromatic setup

### LINKED ISSUE #178: Stories: Determine better setup using Chromatic
Currently we have a top level collection of stories. This was done so we could have a single storybook for all UI components.

Ideally we have a UI storybook and a Design Tokens storybook that are merged together at deploy time so the sidebar nav is like

```
tokens storybook // 1 storybook for all token items
  colors
  etc..
ui storybook
  button // 1 storybook for each UI package
  etc..
```

- [ ] Can we merge storybooks at or before deploy time?
    - This seems possible: https://www.chromatic.com/docs/monorepos#combine-multiple-projects-into-a-single-storybook
    - If we can merge, then we should split the UI stories up by package as NX wants
- [ ] Follow the NX form of adding stories to the components that have existing stories written.
- [ ] Move existing stories into the newly generated storybooks
- [ ] Hook up chromatic to work on PRs
    - https://www.chromatic.com/docs/ci#github-actions

---

- https://www.chromatic.com/docs/storybook

### PR DESCRIPTION
**Pull request by atlwendy**
closes #178

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,6 +86,11 @@
     "//////// Section: Storybook": "==============================",
     "storybook:serve": "start-storybook",
     "storybook:build": "build-storybook -c .storybook -o dist/storybook",
```
Comment (benjamincharity) [inline package.json:88]: Are these two first commands still useful after these changes?

File: .storybook/main.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../stories/**/*.stories.[tj]s'],
+  stories: [STORIES],
```
Comment (benjamincharity) [inline .storybook/main.js:4]: Can we add a comment/note here about the replacement? I could see being _super_ confused   a year from now 😂

**Review by benjamincharity — APPROVED** [review]

File: .storybook/main.js (hunk 2 above)
Comment (atlwendy) [reply .storybook/main.js:4]: Ah okay! I added it in the doc

File: package.json (hunk 1 above)
Comment (atlwendy) [reply package.json:88]: Yes. They are. They're referenced in chromatic command (to build storybook).

**Review by benjamincharity — APPROVED** [review]

**Pull request merged**